BriefAssistant/ClientApp/src/app/components/examples/initial/initial.example.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'initial-example',
+    templateUrl: "./initial.example.component.html",
+    styleUrls: ['./initial.example.component.css']
```

Comment:
I don't understand. None of our files seem to follow that format.

Reply:
Some of them currently do not, but we should stick to a standard instead of each of us following a different naming convention. InitalFormCompnent, InitalWelcomeComponent, and InitalFinalCompoent still need to be fixed. If you would like to fix the other files that don't follow the standard you are welcome to, however since its not part of your PBI you are not required to.